libexec/swiftenv-install

```diff
@@ -128,7 +132,13 @@ install_tar_binary() {
   popd
 
   DESTINATION="$SWIFTENV_ROOT/versions/$VERSION"
-  mv "$INSTALL_TMP/swift-$VERSION_RELEASE"*/ "$DESTINATION"
+  # Some archives don't have an enclosing directory so check "usr" exists first
+  if [ -d "$INSTALL_TMP/usr" ]; then
+    mkdir -p "$DESTINATION"
+    mv "$INSTALL_TMP/usr" "$DESTINATION"
+  else
+    mv "$INSTALL_TMP/swift-$VERSION_RELEASE"*/ "$DESTINATION"
```

Comment:
@kylef Should not be `$VERSION`. `$VERSION_RELEASE` never gets set

Reply:
Yeah looks like that was a bug

Reply:
Not that it really matters as there is only ever one folder in there with the "swift-" prefix.